Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
+MAINTAINER Yi Wang <yi.wang.2005@gmail.com>
+
+COPY build.sh /opt
```

Comment:
BTW, do you know how is volume mount via `-v host:docker` implemented? Does that also cause data transferring?